src/main/java/org/ozwillo/dcexporter/service/DcModelMappingService.java

```diff
@@ -0,0 +1,38 @@
+package org.ozwillo.dcexporter.service;
+
+import org.ozwillo.dcexporter.dao.DcModelMappingRepository;
+import org.ozwillo.dcexporter.dao.SynchronizerAuditLogRepository;
+import org.ozwillo.dcexporter.model.DcModelMapping;
+import org.ozwillo.dcexporter.model.SynchronizerAuditLog;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+@Component
+public class DcModelMappingService {
+
+    @Autowired
+    private DcModelMappingRepository dcModelMappingRepository;
+
+    @Autowired
+    private SynchronizerAuditLogRepository synchronizerAuditLogRepository;
+
+    public List<DcModelMapping> getAllAuditLogWithModel() {
+        List<DcModelMapping> dcModelMappingListWithLog = new ArrayList<DcModelMapping>();
```

Comment:
Not really fan of this.

Would be better to keep model mappings and audit logs separated. For instance, first fetch the model mappings, then for each of them, fetch the audit log.